inc/ticket.class.php

```diff
@@ -830,4 +853,83 @@ static function filter_actors(array $params = []): array {
 
        return $params;
     }
+
+    public static function addToTimeline($options)
+    {
+        $item = $options['item'];
+        $group = new Group_Ticket();
+        $groupUsed = $group->find([
+            'tickets_id' => $options['item']->getID(),
+            'type' => CommonITILActor::ASSIGN,
+        ]);
+        if (empty($groupUsed)) {
+            $groupUsed = 0;
+        } else {
+            $temp = $groupUsed;
+            $groupUsed = [];
+            foreach ($temp as $grp) {
+                $groupUsed[] = $grp['groups_id'];
+            }
+        }
+        $entity = $options['item']->fields['entities_id'];
+        $listEntities = getSonsOf(Entity::getTable(), $entity);
+        if (empty($listEntities)) {
+            $listEntities = $entity;
+        }
+        $criteria = [
+            'OR' => [
+                'AND' => [
+
+                    'entities_id' => $listEntities,
+                    'is_recursive' => 1
+                ],
+                ['entities_id' => $entity],
+            ],
+            'is_assign' => 1,
+            'AND' => [
+                'NOT' => [
+                    'id' => $groupUsed
+                ]
+            ]
+        ];
+        $groupBis = new Group();
+        $groups = $groupBis->find($criteria);
+
+        $itemtypes = [];
+
+        $ticket_task = new PluginEscaladeTicket();
+        if (!empty($groups)) {
+            $itemtypes['escalation'] = [
+                'type' => 'PluginEscaladeTicket',
+                'class' => 'action-escalade',
+                'icon' => 'fas fa-comments',
+                'label' => __('Climb', 'escalade'),
+                'item' => new self()
+            ];
+        }
+
+        return $itemtypes;
+    }
+
+    public function showForm($ID, $options = [])
+    {
+        global $CFG_GLPI;
+        $groupsAssign = new Group_Ticket();
+        $groups = $groupsAssign->find([
+            'tickets_id' => $options["parent"]->getID(),
+            'type' => CommonITILActor::ASSIGN,
+        ]);
+        $used = array();
+        foreach ($groups as $grp) {
+
+            $used[$grp['groups_id']] = $grp['groups_id'];
+
+        }
+        $options['action'] = $CFG_GLPI["root_doc"] . PLUGIN_ESCALADE_DIR_NOFULL . '/front/ticket.form.php';
```

Comment:
```suggestion
        $options['action'] = PLUGIN_ESCALADE_WEBDIR . '/front/ticket.form.php';
```
I think `PLUGIN_ESCALADE_WEBDIR`  already do what you are looking for.